checker.c

```diff
@@ -1,21 +1,77 @@
 #include <stdio.h>
 #include <assert.h>
+#include "checker.h"
+
+int compareOneLimit (float value, float limit){
+    int retval=1;
+    if(value > limit) {
+        printf("out of range!\n");
+        retval = 0;
+    }
+    else {
+        printf("OK!\n");
+    }
+    return retval;
+}
+
+int compareTwoLimits (float value, float minlimit, float uplimit){
```

Comment:
Compare is used to check if the value is same or not which is not the same as value is within range.
Naming of this method can be improved.